sgkit/model.py

```diff
@@ -31,26 +31,26 @@ def create_genotype_call_dataset(
     variant_contig_names
         The contig names.
     variant_contig
-        array_like (element type: int).
+        [array_like, element type: int]
         The (index of the) contig for each variant.
     variant_position
-        array_like (element type: int)
+        [array_like, element type: int]
         The reference position of the variant.
     variant_alleles
-        array_like, zero-terminated bytes, e.g. "S1", or object
+        [array_like, element_type: zero-terminated bytes, e.g. "S1", or object]
         The possible alleles for the variant.
     sample_id
-        array_like (element type: str or object)
+        [array_like, element type: str or object]
         The unique identifier of the sample.
     call_genotype
-        array_like (element type: int) Genotype, encoded as allele values
+        [array_like, element type: int] Genotype, encoded as allele values
         (0 for the reference, 1 for the first allele, 2 for the second allele),
         or -1 to indicate a missing value.
     call_genotype_phased
-        array_like (element type: bool), optional A flag for each call indicating if it is
+        [array_like, element type: bool], optional A flag for each call indicating if it is
```

Comment:
One last change then I think we're there: 

[array_like, element type: bool, optional]

Reply:
Done, thanks!